employee.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+<head>
+  page 2
+  <title>Pat's Salmon Cookies</title>
+  <link rel="stylesheet" type="text/css" href="styles.css"/>
+  <link href="https://fonts.googleapis.com/css?family=Frijole" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Amatic+SC" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <ul>
+      <li><a href="aboutUs.html">About Us<a/></li>
+      <li><a href="locations.html">Locations<a/></li>
+      <li><a href="menu.html">Menu<a/></li>
+      <li><a href="contact.html">Contact<a/></li>
+      <li><a href="employee.html">Employee<a/></li>
+    </ul>
+  </header>
+
+
+
+  <h3>Sales Data</h3>
+  <form action="https://requestb.in/qjtonmqj">
+    <fieldset>
+      <legend>Create New Store Location</legend>
+<!-- I don't want bullet points on my inputs so I am using a <p> tag instead of a <li> tag. -->
+      <p>
+        Location Name: <label for="locName" ></label>
+        <input type="text" name="locationName" placeholder="First &amp Pike"  id="locName"/><br/>
```

Comment:
GitHub is nice about warning you when you have invalid HTML. In this case, you're missing a `;`.